VAMobile/src/constants/linking.tsx

```diff
@@ -44,6 +45,24 @@ export const linking: LinkingOptions<any> = {
           },
         ],
       }
+    } else if (pathParts[0] === 'appointments' && pathParts.length === 2) {
```

Comment:
I'll probably make a CU ticket for implementing a cleaner way to add logic for each deep link, because as the number of deep links grow, this function will get pretty large. Maybe a helper function that can generate the state when passed the screen name and route parameters